fluent/toInterfaceValue_test.go

```diff
@@ -0,0 +1,55 @@
+package fluent_test
+
+import (
+	"testing"
+
+	"github.com/ipld/go-ipld-prime/fluent"
+	"github.com/ipld/go-ipld-prime/node/basicnode"
+)
+
+func TestListValue(t *testing.T) {
+	a := []string{"a", "b", "c"}
+	n, err := fluent.Reflect(basicnode.Prototype.Any, a)
+	if err != nil {
+		t.Fatal(err)
+	}
+	out, err := fluent.ToInterfaceValue(n)
+	if err != nil {
+		t.Fatal(err)
+	}
+	outArr := out.([]interface{})
+
+	if len(a) != len(outArr) {
+		t.Errorf("Mismatch in array size")
+	}
+
+	for i, v := range outArr {
+		if a[i] != v {
+			t.Errorf("expected %v, got %v at index %v", a[i], v, i)
+		}
+	}
+}
+
+func TestMapValue(t *testing.T) {
+	a := map[string]interface{}{"a": "1", "b": int64(2), "c": 3.14}
+	n, err := fluent.Reflect(basicnode.Prototype.Any, a)
+	if err != nil {
+		t.Fatal(err)
+	}
+	out, err := fluent.ToInterfaceValue(n)
+	if err != nil {
+		t.Fatal(err)
+	}
+	outM := out.(map[string]interface{})
+
+	if len(a) != len(outM) {
+		t.Errorf("Mismatch in size")
+	}
+
+	for k, v := range outM {
+		if v != a[k] {
+			t.Errorf("expected %v, got %v at key %v", a[k], v, k)
+		}
```

Comment:
You'd save on keystrokes, and have better errors, if you use the qt testing library :) Like @masih did in PRs like https://github.com/ipld/go-ipld-prime/pull/291.

It's not a must to refactor all the code right now, but we might as well use it in new code.